src/pages/auth/AuthLandingPage.tsx

```
@@ -0,0 +1,6 @@
+type Props = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)